src/client/src/services/executions/__mocks__/executions.ts

```diff
@@ -1,6 +1,8 @@
-let mock: jest.Mock<any, any> = jest.fn()
+import { Mock } from "vitest"
 
-export const __setExecute$ = (input: jest.Mock<any, any>) => {
+let mock: Mock<any, any> = vi.fn()
```

Comment:
to make a tad more readable (and leave it to TS inference) .. can simplify this to
```
let mock = vi.fn()

export const __setExecute$ = (input: Mock) => {
```